File: python/files.py

# * Creates the file if it doesn't exist
file = open("./test.txt", "w")
file.write("I'm writing some text!")
file.close()

# * Doing it this way doesn't need the "close" function
with open("./test.txt") as myFile:
    content = myFile.read()
    print(content.split(" "))

File: javascript/files.js
//! Exception: Semicolon must be added at the end of first line
var fs = require("fs/promises");

(async function () {
  //* Creates the file if it doesn't exist
  await fs.writeFile("./test.txt", "My name is Pedro");

  //* The reading function needs the "utf-8" argument
  var file = (await fs.readFile("./test.txt", "utf-8")).split(" ");
  console.log(file);
})();
